src/Microsoft.AspNetCore.Http.Connections.Common/NegotiateProtocol.cs

```diff
@@ -134,6 +136,8 @@ public static NegotiationResponse ParseResponse(Stream content)
                                             }
                                         }
                                         break;
+                                    case TryWebSocketsPropertyName:
+                                        throw new InvalidOperationException($"It seems like you are trying to connect to an ASP.NET SignalR Server. This client only supports connecting to an ASP.NET Core SignalR Server.");
```

Comment:
I just created https://aka.ms/signalr-core-differences that links to the SignalR Core "differences" doc. Maybe we should put that in the exception message? aka.ms links can be modified at any time, so it's like an FWLink in that we can redirect it when necessary. cc @muratg @Eilon @davidfowl